server/src/jobs/applications/fixApplications.ts

```diff
@@ -0,0 +1,80 @@
+import { logger } from "@/common/logger"
+import { db } from "@/common/mongodb"
+import { asyncForEach } from "@/common/utils/asyncUtils"
+
+const removeOrReplaceCharsInDB = async (collection: string, field: string, charsToReplace: string, replacementChar: string = "") => {
+  logger.info(`remplacement de ${charsToReplace} par ${replacementChar} dans ${collection}.${field}`)
+
+  const charsRegex = new RegExp(`[${charsToReplace}]`, "gi")
+  const applicants = await db
+    .collection(collection)
+    .find({ applicant_email: { $regex: charsRegex } })
+    .toArray()
+
+  applicants.map(async (application) => {
+    console.log(application.applicant_email)
+
+    const correctedEmail = application.applicant_email.replace(charsRegex, "e")
+
+    application.applicant_email = correctedEmail
+
+    await db.collection(collection).save(application)
+  })
+}
```

Comment:
Problèmes de performances ici, on charge tout en RAM. En plus il manque un `await Promise.all`.

```suggestion
  const applicantsCursor = await db
    .collection(collection)
    .find({ applicant_email: { $regex: charsRegex } });

  for await (const application of applicantsCursor) {
    console.log(application.applicant_email)

    const correctedEmail = application.applicant_email.replace(charsRegex, "e")

    application.applicant_email = correctedEmail

    await db.collection(collection).save(application)
  }
}
```